frontend/src/tests/nonprofitApply/nonprofitApplyContent.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
```

Comment:
(nit) can change name to criteriaContent.test.js

Reply:
done